Headers/Foundation/NSObjCRuntime.h

```diff
@@ -286,13 +286,20 @@ DEFINE_BLOCK_TYPE(NSComparator, NSComparisonResult, id, id);
 /**
  * Declare Apple availability macros for compatibility purposes as no-ops.
  */
-#ifndef NS_CLASS_AVAILABLE
-#define NS_CLASS_AVAILABLE(x, y)
-#endif
```

Comment:
I'm only concerned that we lost `#ifndef` guards. Is this intentional and justified? If so I'll merge.